conda/recipes/dask-cuda/meta.yaml

```
@@ -30,6 +30,7 @@ requirements:
     - python
     - pip
     - rapids-build-backend>=0.3.0,<0.4.0.dev0
+    - setuptools
```

```suggestion
    - setuptools>=64.0.0
```

Let's please match the floor in https://github.com/rapidsai/dask-cuda/blob/d70f28d758f2dcae5aa8aeaad759c6acfa65309c/dependencies.yaml#L78

Somewhere around the 64.0.0 release (don't remember exactly where) is where `setuptools` added support for `pyproject.toml`, which we rely on.